install/upgrade_to_2.5.php

```diff
@@ -240,14 +240,11 @@ function updateForm_Profile_2_5(Migration $migration) {
       $migration->migrationOneTable('glpi_plugin_formcreator_forms_profiles');
 
       // fill missing uuid
-      $obj = new PluginFormcreatorForm_Profile();
-      $all_form_profiles = $obj->find("uuid IS NULL");
-      foreach ($all_form_profiles as $form_profiles_id => $form_profile) {
-         $obj->update([
-            'id'   => $form_profiles_id,
-            'uuid' => plugin_formcreator_getUuid()
-         ]);
-      }
+      $DB->update(PluginFormcreatorForm_Profile::getTable(), [
```

Comment:
See previous comment about `DB::update` vs `CommonDBTM::update`

Reply:
same as previous, i think it's not necessary to pass into commondbtm function.
The purpose of this part is to fill missing uuid in the update process.